test/testdata/packager/visibility/foo/__package.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+# typed: strict
+# enable-packager: true
+
+class Foo < PackageSpec
+  visible_to Bar
```

Comment:
Do you think it's worth adding a test where `visible_to` is trying to specify a package that doesn't exist?